### ISSUE TITLE
What happens if the cache size is bigger then the disc size?

### ISSUE THREAD

**Issue by soulflyman** [bntjah/lancache#74]
I have 1TB available for caching. In the 'caches' file are cache sizes defined that are much bigger.

So what happens if the 1TB disk space is full but steam cache is defined with 5TB. Will it stop working and run in errors or does it delete the oldest cache files in the steam cache (or in any cache)?

**Comment by bntjah** [comment]
Well @imac2009 just had this happen to him and it stops serving out any content...
So stops working but I haven't heard anything about any errors...

@imac2009 please confirm?

**Comment by imac2009** [comment]
In case storage is full, lancache isn't serving cached content nor non-cached content. It pretty much leaves you with a scenario like DNS isn't set up correctly.
Like if ppl start a download in steam, they are getting nothing, download doesn't start at all at client side.
On the bright side, nginx is not crashing. You can manually delete content from your caching folders to free space up and nginx will instantly go back to fully operational caching.
tl;dr: get your cache size right :)
